docs/building_the_project.rst

```diff
@@ -490,6 +490,38 @@ A few often-used architectures are listed below:
      - | Radeon Instinct(TM) MI 25 Accelerator
        | Radeon(TM) RX Vega 64/56 Graphics
 
+Building for SYCL-BLAS
+^^^^^^^^^^^^^^^^^^^^^^
+
+Note the SYCL-BLAS backend is experimental and only supports a subset of the
+operations and features for now.
+SYCL-BLAS is used as a header only library.
+The install folder must be generated via cmake with the following commands:
+
+.. code-block:: bash
+
+   # Inside <path to SYCL-BLAS>
+   mkdir build && cd build
+   cmake -DCMAKE_INSTALL_PREFIX=<path to SYCL-BLAS install directory> -DINSTALL_HEADER_ONLY=ON ..
+   cmake --build . --target install
+
+
+.. code-block:: bash
+
+   # Inside <path to onemkl>
+   mkdir build && cd build
+   cmake .. -DENABLE_SYCLBLAS_BACKEND=True                    \
+            [-DREF_BLAS_ROOT=<reference_blas_install_prefix>] \ # required only for testing
+            -Dsycl_blas_DIR=<path to SYCL-BLAS install directory>
```

Comment:
We cannot change `sycl_blas_DIR` to `SYCLBLAS_DIR` since `sycl_blas_DIR` is the name generated by CMake when installing SYCL-BLAS in header-only mode. 

Potentially we could change `ENABLE_SYCLBLAS_BACKEND` to `ENABLE_sycl_blas_BACKEND`. However, this would be inconsistent with other `ENABLE_XXXX_BACKEND` parameters,

Reply:
Okay, I guess this makes sense the way it is.